planning/behavior_velocity_planner/config/intersection.param.yaml

```diff
@@ -15,6 +15,8 @@
       detection_area_length: 200.0 # [m]
       detection_area_angle_threshold: 0.785 # [rad]
       min_predicted_path_confidence: 0.05
+      walkway: 
+        external_input_timeout: 1.0
```

Comment:
@yamazakiTasuku This diff seems to be uncecessary. Could you remove that?

Reply:
And, I think it is better to add this diff to [here](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_velocity_planner/config/crosswalk.param.yaml) instead.